src/service/viewport-impl.js

```diff
@@ -85,6 +85,9 @@ export class Viewport {
     /** @private {?number} */
     this.lastMeasureScrollTop_ = null;
 
+    /** @private {boolean} */
+    this.duringAnimationFrame_ = false;
```

Comment:
Can we rename this to something a bit more descriptive? `scrollAnimationFrameThrottled`?